hw/top_earlgrey/data/ip/chip_spi_device_testplan.hjson

```diff
@@ -26,7 +26,7 @@
         "SPI_DEVICE.HW.LAST_READ_ADDR",
```

Comment:
I'll just point out that the listed tests don't cover the full feature set. There are no reads during bootstrapping.

Reply:
Thanks for pointing this out. I split the test into two test, the new test will be a spi dev flash smoke test.

Reply:
We actually don't have any chip-level tests that exercise this feature. It's also not active in Passthrough mode (only Flash mode, which doesn't do passthrough).

Reply:
I noticed that when I started looking at the passthrought test, I didn't close the [issue](https://github.com/lowRISC/opentitan/issues/20631), so I'll create a spi_device_smoketest to cover that.